.travis.yml

```diff
@@ -31,12 +36,27 @@ before_script:
     # Get last remaining tags, if any.
     - git fetch --tags origin
 
-    - make extracheck
-
-script:
-    - make coverage
-
-    - make
+jobs:
+  include:
+    - stage: build
+      script:
+          - make build # Builds for all three platforms using the go-cross-compiler (is this necessary?)
+          - make extracheck
+      os: linux
+    - stage: build
+      script: make build # Builds for all three platforms using the go-cross-compiler (is this necessary?)
+      os: osx
+    - stage: test
+      install: export PATH=/usr/local/opt/e2fsprogs/sbin/debugfs:$PATH # debugfs is not added to the path by default.
+      script:
+          - make coverage
+          - make test # test & extracheck
```

Comment:
Ambiguous comment: I read this as `make test` = "Run tests and extra check", but I suppose you mean it more as a TODO? I suggest removing it, since the extracheck only needs to be done on one platform (code analysis), and it's already present in the build stage.

Reply:
Yes, sry, should have commented, this is WIP. Will be gone by the end!

Reply:
Meta-comment :)